### ISSUE TITLE
Top of green highlight box separated

### ISSUE THREAD

**Issue by molotov681** [naturalcrit/homebrewery#463]
link to my doc:
http://homebrewery.naturalcrit.com/edit/Hkl9yzKcPb

the second green highlight box is at the top of the second column, but the top of the green highlight box (the shadow if you will) appears at the bottom of the first column, separated from the green highlight box itself. I have tried many things in between but cannot get this shadow to move.

Not a big deal but thought you should be aware.
**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

  # I am the Herbalist
  Herbalism has been around since long before the days of high magic. The most powerful and influential people in these societies were accomplished herbalists. Before magic was harnessed, herbalism provided some of the most potent benevolent boons as well the strongest damaging curses available.

  ### Herbs
  For every hour spent foraging for herbs, and a successful nature or survival check (your choice), with proficiency bonus if you are proficient with the kit, you find 1 GP worth valuable herbs used as components for crafting potions, tonics, salves, and tinctures. These herbs reduce the crafting cost of potions accordingly.

  ### Herbalist Alchemy
  Profiency with the herbalism kit is required to practice herbalist alchemy.  

  The crafting cost of a potion is half the market value of the given potion. This can be reduced with foraged herbs to a cost no less than 3 GP. This cost must be paid to create the potion, and represents the cost of components purchased and used in the creation process.

  Each hour spent crafting potions yields **1 GP times your proficiency bonus** worth of distilled components toward the total creation cost. Each day spent crafting is made with an intelligence check plus your proficiency bonus. A failed check may result in the loss of the componets from that day, as determined by DM.

  Alchemy takes an intricate setup (1 hour) including a fire, a pot, and several containers to transfer and mix liquids, in addition to the items found in the herbalism kit, as well as vials for the final product. While alchemy can be performed in the wild such an act may increase the difficulty of the task or impose disadvantage on the checks; this task is best done in a controlled, urban environment.

  For more difficult (multi-day) brews, each successful day may lower the difficulty of subsequent efforts, as your skills are imporoved and refined, the product you are producing is becoming more pure and easier to work with.

  It is possible the world may provide you with other means of further increasing your abilites in these areas.
  <this is how you make comments!!! woot!>

  >### Potion of Healing
  Market Value: 50 GP  
  Crafting Cost: 25 GP\*  
  Crafting time: 12 hours\*\*  
  <br/>
  **minus the cost of foraged herbs used*
  <Br/>
  ***with a proficiency bonus = 2*

  <BR/>

  >### Healing Poultice
  Market Value: n/a  
  Crafting Cost: 1 GP minus foraged herbs  
  Crafting Time: 1 minute  
  <p>When applied to wounds this poultice helps promote coagulation and healing. As a non-toxic compound it can also be wetted and squeezed to yield a healing liquid.  
  <p>When applied to a creature with 0 hit points, that creature regains 1 hit point. Has no effect on creatures with greater than 0 hit points.  
  <p>Crafting a healing poultice does not require setup and does not incur penalties in the wild.
  <p>A poultice is made of fresh herbs and thus has a variable and short shelf life (1d4-1 days per poultice).

  <br/>  

  >### Herbalism Kit 
  Crafts boon potions, e.g. healing, antitoxin, resistance, etc.  
  ### Alchemy Kit 
  Crafts alteration potions, e.g. climbing, fire breathing, giant strength, etc.  
  ### Poison Kit
  Crafts curse potions, e.g. poison, love, sharpness, etc.  
  <br/>
  >Creating a potion using an alternate kit, or a kit with which you are not proficient, imposses an intelligence check with disadvantage without proficiency bonus for each day of crafting. A failed check may result in the loss of the componets from that day, as determined by DM.
  <br/>  

  <this is a comment
  [Reference: consult your DM](http://i.imgur.com/WhhegbS.png","alchemy potion list)
  >
</pre></code></details>

**Comment by molotov681** [comment]
So the problem solved itself a few hours later. At one point when creating this post I had accidentally highlighted all my code and hit 'tab' and that gave everything a paragraph indent (in the code), and when I found this later and highlighted everything and hit 'shift tab' to remove it, a short while later I noticed that the problem of the separated top shadow was gone. Not sure what happened, but glad it is fixed, it was kinda driving me nuts. Closing ticket.
